docs/fidelity-bonds.md

```diff
@@ -157,7 +157,17 @@ JoinMarket's coin control feature, so before spending you need to unfreeze the
 coins using `python3 wallet-tool.py <walletname> -m 0 freeze`.
 
 Once unfrozen and untimelocked the coins can be spent with a non-coinjoin transaction with
-`sendpayment.py -N 0`. NB You cannot export the private keys (which is always disadvised, anyway)
+`sendpayment.py -N 0`.
+
+It is **not** possible to passively spend these untimelocked coins in your yield generator;
+they are auto frozen while the yield generator runs, because their signing scripts are not
+compatible with the current Joinmarket protocol.
+
+It **is** possible to spend these untimelocked coins in a coinjoin as a taker.
+
+It is not *currently* possible to spend these untimelocked coins in a PSBT or in a Payjoin (which uses PSBT), until our PSBT code is updated to support these custom scripts.
```

Comment:
Spending them in a Payjoin even after adding timelock UTXO support to our PSBT will not make much sense, as for payjoin you want all input scripts to be of the same type. Only exception would be payjoin against another JM user, where both inputs in a final payjoin tx will be expired fidelity bond UTXOs.

Reply:
Oh yes. So that is not interesting enough to mention, agreed, I will remove it.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/981/commits/e7c3c4e07c706bf5feb971dc979b22551438c0a3